### ISSUE TITLE
Somfy Tahoma action groups do not trigger

### ISSUE THREAD

**Issue by Numberone77** [openhab/openhab-distro#1275]
I used Tahoma action groups in OH2.5 and would like to use them in OH3 als well. I currentlc upgraded to OH3.M3 on a openhabian sytem. The action groups currently do not work. I send a sendCommand(ON), but they do not trigger at all.

**Comment by lolodomo** [comment]
@octa22 : this issue was created in the wrong place, it is for you, it is normally not something I touched.

**Comment by lolodomo** [comment]
Can the issue be moved into the correct repo ?

**Comment by octa22** [comment]
Action groups are fixed in 3.0.1

**Comment by Numberone77** [comment]
Yes, it could be moved. BR Matthias --Diese Nachricht wurde von meinem Android Mobiltelefon mit WEB.DE Mail gesendet.Am 08.04.21, 10:50 schrieb lolodomo ***@***.***>:

   
  Can the issue be moved into the correct repo ?

**Comment by Numberone77** [comment]
I use M3 version. I will try downgrade to 3.0.1--Diese Nachricht wurde von meinem Android Mobiltelefon mit WEB.DE Mail gesendet.Am 08.04.21, 10:53 schrieb Ondrej Pecta ***@***.***>:

   
  Action groups are fixed in 3.0.1

**Comment by octa22** [comment]
The 3.1 Milestone has been fixed in this PR https://github.com/openhab/openhab-addons/pull/10488
